### PR TITLE
[pt] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@jorgercosta can you help us and review this automatic pull request, please ? thanks by advance

File: locales/pt/validation-inline.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => 'Este campo pode conter apenas letras, números, traços e o caractere de sublinhado.',
     'alpha_num'            => 'Este campo pode conter apenas letras e números',
     'array'                => 'Este campo deve ser uma matriz.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'Este campo já está ligado.',
```
Comment (jorgercosta) [inline locales/pt/validation-inline.php:23]: I believe it should 'Este campo já está anexado.' just like in locales/pt/validation.php

**Review by jorgercosta — CHANGES_REQUESTED** [review]

File: locales/pt/validation-inline.php (hunk 1 above)
Comment (caouecs) [reply locales/pt/validation-inline.php:23]: the rest is ok for you ?

**Comment by caouecs** [comment]
Obrigado

**Comment by caouecs** [comment]
@jorgercosta the countries in this [page](https://laravel-lang.github.io/lang/statuses/pt.html) are the same in english and in Portuguese ?

**Pull request merged**